Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    fluent-plugin-throttle (0.0.3)
+    fluent-plugin-throttle (0.0.5)
```

Comment:
the gemfile was automatically updated after I built and installed my gem locally